OData/test/System.Web.OData.Test/OData/Routing/ODataPathRouteConstraintTest.cs

```diff
@@ -149,35 +149,6 @@ public void Match_ReturnsTrue_IfODataPathCanBeParsed()
             Assert.Same(_pathHandler, request.ODataProperties().PathHandler);
         }
 
-        [Fact]
```

Comment:
This isn't ideal, but there weren't any other segments that weren't implemented, and it seems impossible to override ODataPathSegmentTranslator to force a not implemented failure